lib/paper_trail/reifier.rb

```diff
@@ -91,14 +91,17 @@ def init_unversioned_attrs(attrs, model)
       #
       # @api private
       def reify_attribute(k, v, model, version)
-        if model.has_attribute?(k)
-          model[k.to_sym] = v
-        elsif model.respond_to?("#{k}=")
-          model.send("#{k}=", v)
-        elsif version.logger
-          version.logger.warn(
-            "Attribute #{k} does not exist on #{version.item_type} (Version id: #{version.id})."
-          )
+        begin
+          if model.has_attribute?(k)
+            model[k.to_sym] = v
+          elsif model.respond_to?("#{k}=")
+              model.send("#{k}=", v)
+          elsif version.logger
+            version.logger.warn(
+              "Attribute #{k} does not exist on #{version.item_type} (Version id: #{version.id})."
+            )
+          end
+        rescue ActiveRecord::ReadonlyAttributeError
```

Comment:
Is it worth logging a warning here, the line above for when there isn't an attribute?

If there isn't, it'd be good have a comment noting when that can happen.

Reply:
Prior to Rails 7.x, active record would silently "fail" when writing to a read only attribute. I felt that paper trail mimicking this behavior (and not logging) would be fine. 

In one production system, logging would generate 43k of log entries.